src/textual/command.py

```diff
@@ -453,10 +498,21 @@ def _on_click(self, event: Click) -> None:
             self.workers.cancel_all()
             self.dismiss()
 
-    def _on_mount(self, _: Mount) -> None:
+    def on_mount(self, _: Mount) -> None:
```

Comment:
Why are we making this public?

Reply:
quells an error about overriding things. I don't think anyone will extend this anyway,

Reply:
Nit-pick: if going public, we can lose the `_: Mount`.